patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1359,12 +1359,21 @@
 +        return false;
 +    }
 +
++    /**
++     * Set the PathNodeType for this block. null for vanilla behavior.
++     * @return the PathNodeType
++     */
++    @Nullable public net.minecraft.pathfinding.PathNodeType getAiPathNodeType(IBlockState state, IBlockAccess world, BlockPos pos)
```

Comment:
I assume diesieben just wanted to minimize the patch size by putting it all in the same line. A similar thing can be found [here in commit 150566d](https://github.com/diesieben07/MinecraftForge/commit/150566d9a34005e4ec7b91ab0ab99b3fd27d555e#diff-97673260fb0239af80fa467c2f5f5086R1225).